test/functional/http/post/1.second.secret.js

```diff
@@ -0,0 +1,456 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('../../shared');
+var constants = require('../../../../helpers/constants');
+
+var sendTransactionPromise = require('../../../common/apiHelpers').sendTransactionPromise;
+var creditAccountPromise = require('../../../common/apiHelpers').creditAccountPromise;
+var sendSignaturePromise = require('../../../common/apiHelpers').sendSignaturePromise;
+
+var onNewBlockPromise = node.Promise.promisify(node.onNewBlock);
+
+describe('POST /api/transactions (type 1) register second secret', function () {
+
+	var badTransactions = [];
```

Comment:
valid & invalid

Reply:
It is already in place on `0.transfer.js`. We agreed in the past.